nf_core/pipelines/bump_version.py

```diff
@@ -288,7 +289,7 @@ def update_text_file(fn: Path, patterns: List[Tuple[str, str]], required: bool):
             updated = True
             log.info(f"Updated version in '{fn}'")
             log.debug(f"Replaced pattern '{pattern}' with '{replacement}' {count} times")
-        elif required:
+        else:
```

Comment:
won't this show the error when the version is not found? even if required is false

Reply:
nope, because we check `required` in `handle_error` below and it only raises an error if it is `required` otherwise it just does `log.info()`